tcms/testplans/static/testplans/js/search.js

```diff
@@ -9,6 +9,7 @@ function pre_process_data(data) {
 
 $(document).ready(function() {
     var table = $("#resultsTable").DataTable({
+        "pageLength": 100,
```

Comment:
remove the quotes around `pageLength` for consistency with the rest of the code in this file.

Reply:
@schwarzkrieger Was it helpful the session last Thu after the meeting?

Reply:
> 
> 
> @schwarzkrieger Was it helpful the session last Thu after the meeting?

Yes, the session was extremely helpful. Thank you for your time, patience and guidance!!!